File: app/src/main/java/android/mb00/codehub/api/model/Issue.java
package android.mb00.codehub.api.model;

import com.google.gson.annotations.SerializedName;


public class Issue {

    @SerializedName("state")
    private String state;
    @SerializedName("number")
    private String number;
    @SerializedName("title")
    private String title;
    @SerializedName("created_at")
    private String openDate;
    @SerializedName("closed_at")
    private String closeDate;

    public String getState() {
        return state;
    }

    public String getNumber() {
        return number;
    }

    public String getTitle() {
        return title;
    }

    public String getOpenDate() {
        return openDate;
    }

    public String getCloseDate() {
        return closeDate;
    }
}

File: app/src/main/java/mb00/android/codehub/api/model/Issue.kt
package mb00.android.codehub.api.model

import com.google.gson.annotations.SerializedName


data class Issue(
        @SerializedName("state") var state: String?,
        @SerializedName("number") var number: String?,
        @SerializedName("title") var title: String?,
        @SerializedName("created_at") var openDate: String?,
        @SerializedName("closed_at") var closeDate: String?
)
